lighthouse/messages/message.py

```diff
@@ -1,27 +1,32 @@
 import json
 
+from lighthouse.types import EventMessage
+
 
 class Message:
     """Creates a message with the correct payload structure to send to the warehouse."""
 
-    def __init__(self, message=None):
-        self.message = message
+    def __init__(self, message: EventMessage):
+        self._message = message
+
+    @property
+    def message(self) -> EventMessage:
+        return self._message
 
     def payload(self) -> str:
         """Generates the JSON payload of the message.
 
         Returns:
             {str} -- The message JSON payload.
         """
-        return json.dumps(self.message)
+        return json.dumps(self._message)
```

Comment:
Because message now is property we can still use self.message

Reply:
Yes, I will fix this!